scene/main/viewport.cpp

```diff
@@ -1457,8 +1457,6 @@ void Viewport::_gui_show_tooltip() {
 	}
 
 	Control *rp = which; //->get_root_parent_control();
-	if (!rp)
```

Comment:
Please also remove the commented code right above for which this null check existed.